scripts/check_known_pulsars.py

```diff
@@ -408,6 +418,10 @@ def beamform_and_fold(obsid, DI_dir, cal_obs, args, psrbeg, psrend,
     vcstools_version: string
         OPTIONAL - The version of vcstools to use. Default = 'master'
     """
+    base_dir = "{0}/{1}/dpp_pointings/".format(product_dir, obsid)
```

Comment:
should use os.path.join when making paths to avoid extra forward slashes

Reply:
Done